contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -770,7 +770,7 @@ func (s *schedulingPlugin) scheduleOne() {
 	dest, err := s.config.Algorithm.Schedule(pod, s.config.MinionLister) // call kubeScheduler.Schedule
 	if err != nil {
 		log.V(1).Infof("Failed to schedule: %v", pod)
-		s.config.Recorder.Eventf(pod, "failedScheduling", "Error scheduling: %v", err)
+		s.config.Recorder.Eventf(pod, "FailedScheduling", "Error scheduling: %v", err)
```

Comment:
@vishh Do you mean all the components in kubernetes? If so, #12516 and this will cover all the components. Or some components outside kubernetes?